### PR TITLE
ruleset: Stamus Networks Lateral Movement

### PR DESCRIPTION
**Pull request by jasonish**
- ruleset: Stamus Networks lateral movement

### CODE REVIEW

File: index.yaml [hunk 1]
```diff
@@ -132,8 +132,19 @@ sources:
     homepage: https://raw-data.gitlab.io/post/malsilo_2.1/
     checksum: true
 
+  stamus/lateral:
```
Comment (jasonish) [inline index.yaml:135]: As this seems specific to Windows, should that be reflected in the name? If the plan is maybe to be less specific than Windows at some point in the future, best not to embed Windows in the naming.

File: index.yaml [hunk 2]
```diff
@@ -132,8 +132,19 @@ sources:
...
+    summary: Lateral movement rules
+    description: |
+      Suricata ruleset specifically focused on detecting lateral
+      movement in Microsoft Windows environments by Stamus Networks
+    vendor: Stamus Networks
+    min-version: 6.0.6
+    license: GPLv3
+    support-url: https://discord.com/channels/911231224448712714/911238451842666546
+    url: https://ti.stamus-networks.io/open/stamus-lateral-rules.tar.gz
+    checksum: https://ti.stamus-networks.io/open/stamus-lateral-rules-checksum.txt
```
Comment (jasonish) [inline index.yaml]: This form of specifying a checksum URL does not work yet (https://redmine.openinfosecfoundation.org/issues/5684), @pevma, can you create the checksum URL at the rule URL + `.md5`?

File: index.yaml [hunk 3]
```diff
@@ -132,8 +132,19 @@ sources:
...
     checksum: true
 
+  stamus/lateral:
+    summary: Lateral movement rules
+    description: |
+      Suricata ruleset specifically focused on detecting lateral
+      movement in Microsoft Windows environments by Stamus Networks
+    vendor: Stamus Networks
+    min-version: 6.0.6
+    license: GPLv3
```
Comment (jasonish) [inline index.yaml]: License confusion.  The `LICENSE` file is GPLv3, however the rules file is MIT license. If sticking with the GPLv3, also note the difference between `GPL-3.0-only` and `GPL-3.0-or-later` as I'd like to move to SPDX license identifiers in this index.

File: index.yaml (hunk 2 above)
Comment (pevma) [reply index.yaml]: Fixed

File: index.yaml (hunk 3 above)
Comment (pevma) [reply index.yaml]: fixed

**Pull request merged**